src/auth/index.js

```diff
@@ -0,0 +1,127 @@
+import Vue from "vue";
```

Comment:
Are you sure you want to use two spaces? I don't mind but remember to be consistent in other files.

Reply:
These, functions etc... is generated from auth0, so I think is okay?

Reply:
As I said I don't mind (even tho I think four spaces are better) but use two spaces in other files as well